config/routes.rb

```diff
@@ -12,5 +12,10 @@
 
   resources :searches
 
+  get '/images', to: 'searches#images'
+  get '/articles', to: 'searches#articles'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.